app/assets/javascripts/discourse/app/components/topic-list/list.gjs

```diff
@@ -213,12 +213,19 @@ export default class TopicList extends Component {
       <PluginOutlet
         @name="before-topic-list-body"
         @outletArgs={{hash
+          columns=this.columns
```

Comment:
I don't think we should pass this. It exposes a massive amount of core's implementation as 'public API', and encourages people to re-implement core's topic list in their own plugin/theme. That's essentially the same as a template override, and will carry a lot of the same maintenance issues.